.github/workflows/dlp-pipelines.yml

```diff
@@ -117,6 +122,7 @@ jobs:
             else
               sleep 30s
             fi
+            echo "Got number of tables in BQ with id ${{env.INSPECTION_TABLE_ID}}: $table_count ."
```

Comment:
This statement looks unnecessary. Was it for debugging purpose only?

Reply:
On failure, these statements help understand how many findings we are actually getting and why the pipeline is taking longer. Accordingly, PR author can cancel the pipeline beforehand instead of waiting for 30 minutes timeout. This log will guide both in traceability and debugging.